css-three-column-layout/styles.css

```diff
@@ -0,0 +1,71 @@
+.container {
+  margin: 0 auto;
+  max-width: 1100px;
+  font-family: Lato, sans-serif;
+  font-style: normal;
+  font-weight: 400;
+}
+
+.row {
+  display: flex;
+  min-height: 150px;
+  align-items: center;
+  justify-content: center;
+}
+
+.column-full {
+  width: 100%;
+}
+
+.column-one-third {
+  width: calc(100% / 3);
+  flex-basis: 30%;
```

Comment:
if you are going to use `width` I would remove the flex basis here.

Reply:
![css-three-column](https://user-images.githubusercontent.com/106547566/186775003-5e34a3f2-9ff0-4386-924b-d4b0fcc7f1a1.PNG)